python/cugraph/cugraph/structure/graph_implementation/simpleDistributedGraph.py

```diff
@@ -979,6 +980,83 @@ def convert_to_cudf(cp_arrays):
 
         return ddf
 
+    def decompress_to_edgelist(
+        self,
+        original: bool = True
+    ) -> dask_cudf.DataFrame:
+        """
+        Extract a the edgelist from a graph.
+
+        Parameters
+        ----------
+        original    : bool (default=True)
+                    Flag determining whether to return the original input edgelist
+                    if 'True' or the renumbered one of 'False' and the edgelist was
+                    renumbered.
+
+        Returns
+        -------
+        ego_edge_lists : dask_cudf.DataFrame
```

Comment:
Is this the right name for the return value?  Also, should it be plural?  I figured it would be just `edgelist`